src/test/resources/restores/project_role_admin.xml

```diff
@@ -0,0 +1,4496 @@
+<OpenRosaResponse xmlns="http://openrosa.org/http/response">
+<message nature="ota_restore_success">Successfully restored account vl1!</message>
+<Sync xmlns="http://commcarehq.org/sync">
+    <restore_id>9f7c4e0a3a8ca4e2ecf54444fb7e70cf</restore_id>
+</Sync>
+<Registration xmlns="http://openrosa.org/user/registration">
```

Comment:
Can we trim the file to only contain what's necessary for the test. I believe you should be able to remove most of the info in this file for the assertion test.

Reply:
restore updated here https://github.com/dimagi/formplayer/pull/1411/commits/0663863504d0e7c8eae118273b59572bcf3dc3c5